client/packages/common/src/intl/locales/en/distribution.json

```diff
@@ -35,7 +35,6 @@
   "heading.supply-to-requested": "Set supply to requested",
   "heading.supply-to-approved": "Set supply to approved",
   "heading.tax": "Tax",
-  "heading.edit-tax-rate": "Edit tax rate percentage",
```

Comment:
mad nit but I think if tax rate is a common component, the translation should be in common rather than replenishment 👀

Reply:
I was thinking about that...